exonum_launcher/launch_state.py

```diff
@@ -0,0 +1,50 @@
+"""Launch process state module"""
+from typing import Dict, List
+
+from .action_result import ActionResult
+from .configuration import Artifact, Instance
+
+
+class LaunchState:
+    """State of the deploy&init process."""
+
+    def __init__(self) -> None:
+        self._pending_deployments: Dict[Artifact, List[str]] = {}
+        self._pending_initializations: Dict[Instance, List[str]] = {}
+        self._completed_deployments: Dict[Artifact, ActionResult] = dict()
+        self._completed_initializations: Dict[Instance, ActionResult] = dict()
```

Comment:
Just out of curiosity. Why do you use different ways to initialise a dictionary here ?

Reply:
Because sometimes I write code and don't look at the lines that surround it ^_^"
I'll fix that in the next PR.